app/views/sign_in_security_check_failed/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<% self.title = t('security_check_failed.title') %>
+
+<%= render StatusPageComponent.new(status: :warning) do |c| %>
+  <% c.with_header { t('security_check_failed.title') } %>
+  <p>
+    <%= t('security_check_failed.details') %>
+    <%= new_tab_link_to(
+          t('security_check_failed.learn_more', app_name: APP_NAME),
+          help_center_redirect_path(
+            category: 'trouble-signing-in',
+            article: 'security-check-failed',
+          ),
+        ) %>
+  </p>
+
+  <ul class="usa-list">
+    <li><%= t('security_check_failed.info_p1', app_name: APP_NAME) %></li>
+    <li><%= t('security_check_failed.info_p2_html', piv_cac_url: login_piv_cac_url) %></li> 
+    <li><%= t('security_check_failed.info_p3') %></li> 
+  </ul>
+
+  <%= c.with_troubleshooting_options.new do |tc| %>
```

Comment:
The `.new` here isn't necessary and causes an error. Also, `<%` should be used instead of `<%=` for ViewComponent slots, else it'll render twice.

```suggestion
  <% c.with_troubleshooting_options do |tc| %>
```